.tav.yml

```diff
@@ -128,3 +128,8 @@ hapi-async-await:
   node: '>=8.2'
   versions: '^17.0.0'
   commands: node test/instrumentation/modules/hapi.js
+
+cassandra-driver:
+  versions: '>3.1.0 <4'
```

Comment:
From the tav tests it looks like 3.1.x isn't supported.

You can run `./node_modules/.bin/tav cassandra-driver ^3.1.0 --compat -- node test/instrumentation/modules/cassandra/cassandra.js` locally to get an overview of what's supported.